tests/unit/test_awsrequest.py

```diff
@@ -97,83 +97,6 @@ def tell(self):
         return self._stream.tell()
 
 
-class TestAWSRequest(unittest.TestCase):
-
-    def setUp(self):
-        self.tempdir = tempfile.mkdtemp()
-        self.request = AWSRequest(url='http://example.com')
-        self.prepared_request = self.request.prepare()
-        self.filename = os.path.join(self.tempdir, 'foo')
-
-    def tearDown(self):
-        shutil.rmtree(self.tempdir)
-
-    def test_should_reset_stream(self):
```

Comment:
Why is this test being removed?  It's still a method on the aws request object so it seems like we should still be testing it.

Reply:
These tests were for `reset_stream_on_redirect` method on the `AWSPreparedRequest` class that's been removed (since we aren't relying on / hooking into redirects from our client). I'll see if I can rework some of these tests to test `reset_stream` directly without relying on `reset_stream_on_redirect` calling it.